core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java

```diff
@@ -368,6 +369,7 @@ public synchronized void resume() {
     if (mCatchupThread != null && mCatchupThread.isAlive()) {
       mCatchupThread.cancel();
       mCatchupThread.waitTermination();
+      mCatchupThread = null;
```

Comment:
Explicitly setting this to `null` will help us induce the state. `null` means the thread should not be there. `non-null` and `thread.crashed()` means the thread is terminated by an exception. This will help external monitors.